pkg/expression/builtin_time_vec.go

```diff
@@ -547,7 +547,7 @@ func (b *builtinNowWithArgSig) vecEvalTime(ctx EvalContext, input *chunk.Chunk,
 		fsp := 0
 		if !bufFsp.IsNull(i) {
 			if fsps[i] > int64(types.MaxFsp) {
-				return errors.Errorf("Too-big precision %v specified for 'now'. Maximum is %v", fsps[i], types.MaxFsp)
```

Comment:
There is a typos error of mine, so I change it from `fsp` to `fsps[i]` and then merge it with previos commit.